### DISCUSSION TITLE
image is not existing/readable as stated in log file

### DISCUSSION THREAD

**Discussion by alwadey** [davidfrantz/force#323] [Q&A] [answered]
Dera community,
please can you help:
running :
docker run -t -v /home/alwadey/force:/home/alwadey/force davidfrantz/force force-level2 /home/alwadey/force/param/short.prm
 generate in the screen :
 1 images enqueued. Start processing with 1 CPUs

Computers / CPU cores / Max jobs to run
1:local / 8 / 1

Computer:jobs running/jobs completed/%of started jobs/Average seconds to complete
ETA: 0s Left: 0 AVG: 0.00s  local:0/1/100%/1.0s 

in the logfile:
/opt/data/T38RPN/S2A_MSIL1C_20220609T072621_N0400_R049_T38RPN_20220609T085326.SAFE
: /opt/data/T38RPN/S2A_MSIL1C_20220609T072621_N0400_R049_T38RPN_20220609T085326.SAFE
 is not existing/readable

the queue.txt file  is as :
/opt/data/T38RPN/S2A_MSIL1C_20220609T072621_N0400_R049_T38RPN_20220609T085326.SAFE QUEUED
where there is no full path since I downloaded the data.

I am using FORCE v. 3.7.12 as  a docker image in Ubuntu 20.04

thanks.

**Comment by davidfrantz** [comment, marked as answer]
Dear @alwadey ,

This is a question about Docker usage.

You need to mount every folder you need via the -v options. 

The images referenced in the queue are not available as no mount-pair is specified.

Best,
David

**Reply by alwadey** [reply]
Dear @davidfrantz,
thank you so much.
 it was a docker issue as you stated>
